### DISCUSSION TITLE
Composer dependencies require a PHP version >= 8.0.2

### DISCUSSION THREAD

**Discussion by bkpro61** [wintercms/winter#820] [General]
I can't install winter.
nginx error.log message:
FastCGI sent in stderr: "PHP message: PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.2". You are running 7.4.33. 

but system Requirements PHP version 7.2 or higher. Why message PHP 8.0.2?

**Comment by bennothommo** [comment]
@bkpro61 Are you using the Winter web installer? This installs Winter 1.2 at the moment, which requires at least PHP 8.0.2.

If you want to install Winter 1.1, which does support PHP 7.4, you can use this older version of the Web Installer:
https://github.com/wintercms/web-installer/releases/download/v1.0.2/install.zip

Note that besides security fixes, Winter 1.1 won't be getting any more updates, and also, PHP 7.4 is now no longer receiving security updates either.

**Reply by bkpro61** [reply]
I used the installer from the main page of the site next to which it says: "Simply download and extract a Zip file on your server, navigate to the install URL...". It is also written there System Requirements PHP version 7.2 or higher, so it's misleading. There is PHP 8.0.2 not a single word about it.
Thank you for your reply.

**Reply by LukeTowers** [reply]
We're currently working on a major refresh to our website, but I've gone in and updated that page to reflect the latest installation requirements. Thanks for bringing it to our attention!
